### PR TITLE
Add translation to Brazilian Portuguese

### PR DESCRIPTION
**Pull request by chriexpe**

### CODE REVIEW

File: locales/pt_BR.json [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+          "footer2": "Links úteis",
+          "curseforge": "Iris no CurseForge",
+          "modrinth": "Iris no Modrinth",
+          "github": "GitHub",
+          "discord": "Discord",
+          "patreon": "Patreon",
+          "copyright": "© Copyright 2021 IrisShaders. Todos os direitos reservados."
+        },
+        {
+          downloadiris": "Baixe agora - IrisShaders",
```
Comment (thedocruby) [inline locales/pt_BR.json:94]: Missing a quotation mark at the beginning

**Review by thedocruby — CHANGES_REQUESTED** [review]
Translations look okay, but the format of the .json files has been updated since you started working on Portuguese. Please take a look at locales/en.json fro the correct format. Also, you need to add your lang code to js/lang.js

File: locales/pt_BR.json (hunk 1 above)
Comment (chriexpe) [reply locales/pt_BR.json]: Tnks, fixed

**Comment by thedocruby** [comment]
Thanks!

**Pull request merged**